### ISSUE TITLE
Enhancement : add jackson-datatype-jdk8 dependency

### ISSUE THREAD

**Issue by fraf** [jhipster/generator-jhipster#3258]
Is it planned to add this dependency in pom.xml ? :

```
<dependency>
	<groupId>com.fasterxml.jackson.datatype</groupId>
	<artifactId>jackson-datatype-jdk8</artifactId>
	<version>2.7.3</version>
</dependency>
```

It allows to map Json empty/null String to an Optional java 8 type in DTO object.

Could it be useful ? I don't have it with v2.27.1 generator.

**Comment by gmarziou** [comment]
We've just released 3.0, so it is very unlikely that we add this to 2.x branch.

**Comment by cbornet** [comment]
To complement, I think Optional on class attributes is bad practice : it doesn't add any value and it is not serializable. Optional should only be used on method's return types.
